src/qforte/abc/algorithm.py

```diff
@@ -108,6 +108,9 @@ def __init__(self,
         except AttributeError:
             self._hf_energy = 0.0
 
+        if hasattr(system, 'frozen_core_energy'):
+            self._frozen_core_energy = system.frozen_core_energy
```

Comment:
I'd rather keep this information on the molecule, but not put it on the algorithm.